docs/src/featureToggle.js

```diff
@@ -1,7 +1,7 @@
 // need to use commonjs export so that docs/packages/markdown can use
 module.exports = {
   nav_products: true,
-  enable_website_banner: false,
+  enable_website_banner: true,
```

Comment:
Would it make sense to split the top banner and the side nav toc?

<img width="404" alt="Screenshot 2022-02-14 at 14 36 36" src="https://user-images.githubusercontent.com/3165635/153874507-a9fb0406-339e-493c-9d18-76c28655493b.png">

https://deploy-preview-31076--material-ui.netlify.app/getting-started/example-projects/

Reply:
done!